docs/source_files/pixi_tomls/tasks_short_circuit_basic.toml

```diff
@@ -0,0 +1,16 @@
+[workspace]
+channels = ["conda-forge"]
+name = "tasks-short-circuit-basic"
+
+# --8<-- [start:tasks]
+[tasks]
+build = "make build"
+test = "make test"
+
+# Short circuit composition - define a task as an array of dependencies
+all = [{ task = "build" }, { task = "test" }]
+
+# This is equivalent to:
+# all = { depends-on = [{ task = "build" }, { task = "test" }] }
```

Comment:
Let's uncomment this and just give the two tasks slightly different names.

Like `all-short` and `all-long`